app/models/reports/report.rb

```diff
@@ -33,7 +33,7 @@ def path
 	end
 		
 	def container_path
-		File.join(Rails.root, "tmp", "reports", self.id.to_s)
+		File.join(Rails.root, "tmp", "reports", "tenant-#{Apartment::tenant.current}", self.id.to_s)
```

Comment:
Line is too long. [94/80]<br>Use 2 (not 1) spaces for indentation.<br>Redundant `self` detected.<br>Tab detected.